src/api/ApiSpec.php

```diff
@@ -47,6 +48,15 @@ class ApiSpec {
      * @var array
      */
     private $functionArgs = array(
+        'actTourn' => array(
+            'mandatory' => array(
+                'tourn' => 'number',
+                'action' => 'alnum',
+            ),
+            'permitted' => array(
+                'button_names' => 'string',
```

Comment:
To fix: everything in the API spec is camel-case, including attribute names.

Reply:
Fixed